ocaml/lambda/matching.mli

```diff
@@ -19,16 +19,40 @@ open Typedtree
 open Lambda
 open Debuginfo.Scoped_location
 
+(* An action is a right-hand side of a case (as in a match, function, try) *)
+type action =
```

Comment:
We could consider making this type abstract. Users of this module should never need to inspect the shape of a value of type `action`. Can we just expose ways of constructing these values?

Reply:
Done! Let me know if the new documentation is unclear